jkube-kit/build/api/src/main/java/org/eclipse/jkube/kit/build/api/auth/RegistryAuth.java

```diff
@@ -32,69 +33,69 @@
 @EqualsAndHashCode
 public class RegistryAuth {
 
-    public static final RegistryAuth EMPTY_REGISTRY_AUTH =
```

Comment:
This class was completely reformatted and is really hard to track the changes (at least in the GH interface).
Could you please revert the reformatting so that it's clearer for us to see what was changed here?